typings/index.d.ts

```diff
@@ -1354,6 +1354,12 @@ declare module "noblox.js" {
     */
     function getPlayerBadges(userId: number, limit?: Limit, cursor?: string, sortOrder?: SortOrder): Promise<PlayerBadges>
 
+    /**
+     * Gets aa users groups
+     */
+    function getGroups(userId: number): Promise<Group[]>;
```

Comment:
It actually doesn't return the GroupInterface. It's rather similar but it has some differences, see below:
![image](https://user-images.githubusercontent.com/12759332/98504845-fb4a0a80-2225-11eb-8ccb-456f530c7cb9.png). You might want to add a new interface for this.

Reply:
Yep - I realised this a bit late on. I've done some modifications locally and I'm testing them, then I'll commit n' push. Stand-by!